client/state/sites/features/actions.js

```diff
@@ -46,6 +50,44 @@ export function fetchSiteFeatures( siteId ) {
 	};
 }
 
+export function fetchJetpackSitesFeatures() {
+	return ( dispatch ) => {
+		dispatch( {
+			type: JETPACK_SITES_FEATURES_FETCH,
+		} );
+
+		return wpcom.req
+			.get( `/me/sites/features` )
+			.then( ( data ) => {
+				const features = {};
+				for ( const siteId in data.features ) {
+					features[ siteId ] = createSiteFeaturesObject( data.features[ siteId ] );
+				}
+
+				dispatch( fetchJetpackSitesFeaturesReceive( features ) );
+
+				dispatch( {
+					type: JETPACK_SITES_FEATURES_FETCH_SUCCESS,
+				} );
+			} )
+			.catch( ( error ) => {
+				const errorMessage =
+					error.message ||
+					i18n.translate(
+						'There was a problem fetching site features. Please try again later or contact support.'
```

Comment:
In this case I'm just re-using the behavior from the single site features endpoint so I don't want to do something like refresh. Ideally we would redo all of this in react-redux so that it does handle retries.